sync/gh.py

```diff
@@ -82,6 +82,16 @@ def add_labels(self, pr_id, *labels):
         issue = self.repo.get_issue(pr_id)
         issue.add_to_labels(*labels)
 
+    def remove_labels(self, pr_id, *labels):
+        issue = self.repo.get_issue(pr_id)
+        for label in labels:
+            self.remove_label(pr_id, label, issue=issue)
+
+    def remove_label(self, pr_id, label, issue=None):
```

Comment:
I'm quite tempted to just inline this into `remove_label`, since `remove_labels` handles the case of a single label with just the same API.

Reply:
I've inline the methods into a single `remove_labels`